server/index.ts

```diff
@@ -118,9 +131,22 @@ async function handleConnect(player: alt.Player) {
     Rebar.player.useNative(player).invoke('triggerScreenblurFadeIn', 1000);
 }
 
+async function handleDisconnect(player: alt.Player) {
+    const account = Rebar.document.account.useAccount(player);
```

Comment:
This may not always work. The player can sometimes be marked invalid by the time it gets to this event.

You can often get the `id` before the player object goes invalid.

Reply:
Yep, that was to bypass an error in the console when the player was kicked without account._id set. It doesn't matter anyway now. Fixed.